commands/allTheTest.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
This package isn't an appropriate place for scripts. If this is worth sharing, it should be documented and go in scripts. Otherwise it should stay out of the repo.

Reply:
This was not supposed to be pushed, was helpful to debug some failure, will be removing

Reply:
git add . is bad for you.

Reply:
`git alias au="add -u"`. so much better